src/components/button/button.jsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import React from 'react';
+import { DownOutlined } from '@ant-design/icons';
+import { Dropdown, message, Space } from 'antd';
+
+const onClick = ({ key }) => {
+  message.info(`Click on item ${key}`);
+};
+
+const Button = ({ open }) => {
```

Comment:
This doesn't look like a button component, could you rename this?

Reply:
I am actually not using this again too, it is one of the earliest components I created, I should have deleted but kep postponing since it was not affecting the other stuff I was working on